vespalib/src/vespa/vespalib/regex/regex.cpp

```diff
@@ -49,6 +49,16 @@ class Regex::Impl {
         }
         return RE2::FullMatch(StringPiece(input.data(), input.size()), _regex);
     }
+
+    std::pair<std::string, std::string> possible_anchored_match_prefix_range() const {
+        constexpr int max_len = 128; // TODO determine a "reasonable" value. RE2 docs are not clear on this.
+        std::string min_prefix, max_prefix;
+
+        if (!_regex.PossibleMatchRange(&min_prefix, &max_prefix, max_len)) {
+            return {};
```

Comment:
Consider adding a test case for this code path as well.

Reply:
The easiest way to deterministically get this to actually fail is by supplying a bad regex, so I'll add a test with that.